src/pages/settings/Security/TwoFactorAuth/TwoFactorAuthSteps.js

```diff
@@ -1,77 +0,0 @@
-import React, {useCallback, useEffect, useState} from 'react';
-import {withOnyx} from 'react-native-onyx';
-import CodesStep from './Steps/CodesStep';
-import DisabledStep from './Steps/DisabledStep';
-import EnabledStep from './Steps/EnabledStep';
-import VerifyStep from './Steps/VerifyStep';
-import SuccessStep from './Steps/SuccessStep';
-import ONYXKEYS from '../../../../ONYXKEYS';
-import CONST from '../../../../CONST';
-import * as TwoFactorAuthActions from '../../../../libs/actions/TwoFactorAuthActions';
-import TwoFactorAuthContext from './TwoFactorAuthContext';
-import {defaultAccount, TwoFactorAuthPropTypes} from './TwoFactorAuthPropTypes';
-import useAnimatedStepContext from '../../../../components/AnimatedStep/useAnimatedStepContext';
-
-function TwoFactorAuthSteps({account = defaultAccount}) {
-    const [currentStep, setCurrentStep] = useState(CONST.TWO_FACTOR_AUTH_STEPS.CODES);
-
-    const {setAnimationDirection} = useAnimatedStepContext();
-
-    useEffect(() => () => TwoFactorAuthActions.clearTwoFactorAuthData(), []);
-
-    useEffect(() => {
-        if (account.twoFactorAuthStep) {
-            setCurrentStep(account.twoFactorAuthStep);
-            return;
-        }
```

Comment:
Didn't we miss this redirect? I can't find similar logic after your commits

Reply:
Hmm, I'll have to add it back so we can stop user from directly accessing links.